server/services/interventionsService.ts

```diff
@@ -0,0 +1,29 @@
+import RestClient from '../data/restClient'
+import logger from '../../log'
+import { ApiConfig } from '../config'
+
+interface Referral {
+  id: string
+  desiredOutcomes: DesiredOutcome[]
+}
+
+interface DesiredOutcome {
+  id: string
+  text: string
```

Comment:
would prefer more descriptive field name - maybe `description`?

Reply:
also, do these need IDs? i guess all these questions are not super relevant for this PR. we can discuss in a different forum.

Reply:
Have removed desired outcomes for now.